app/scripts/modules/core/src/presentation/forms/fields/renderContent.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+
+export function renderContent<T>(Content: string | JSX.Element | React.ComponentType<T>, props: T) {
+  if (typeof Content === 'function' && Content.prototype && Content.prototype.isReactComponent) {
```

Comment:
guessing you already checked, but does this catch `forwardRef` components?

Reply:
no I didn't check.  I already forgot what forwardRef does.  will check

Reply:
not sure that this needs forwardRef support 🤔 
If we end up needing it we can add then